python.d/ceph.chart.py

```diff
@@ -0,0 +1,165 @@
+# -*- coding: utf-8 -*-
+# Description: ceph netdata python.d module
+# Author: Luis Eduardo (lets00)
+
+try:
+    import rados
+    CEPH = True
+except ImportError:
+    CEPH = False
+
+import json
+from bases.FrameworkServices.SimpleService import SimpleService
+
+# default module values (can be overridden per job in `config`)
+update_every = 10
+priority = 60000
+retries = 60
+
+ORDER = ['general', 'pool_usage', 'pool_objects', 'osd_usage']
+
+CHARTS = {
+    'general': {
+        'options': [None, 'Ceph General Options', 'KB', 'general', 'ceph.general', 'stacked'],
+        'lines': [
+            ['general_usage', 'Usage', 'absolute', 1, 1024],
+            ['general_available', 'Available', 'absolute', 1, 1024]
```

Comment:
Dimension order is a thing. 

> This logic currently says that colors are in pairs: positive, negative, positive, negative, etc. Positive can be anything (green, blue, bright yellow, etc), but negative have more red in them (red, brown, dark purple, etc).

`Available` should be the first (green). And please make it lowercase.
like
`avail`
`used`